packages/loopring_v3/contracts/core/impl/DefaultDepositContract.sol

```diff
@@ -95,11 +95,9 @@ contract DefaultDepositContract is IDepositContract, Claimable
         returns (uint96 amountReceived)
     {
         if (isETHInternal(token)) {
-            require(msg.value == amount, "INVALID_ETH_DEPOSIT");
+            require(msg.value >= amount, "INVALID_ETH_DEPOSIT");
```

Comment:
Any reason why `>=` now?

Reply:
We still allow users to tip the operator for faster deposit processing.

Reply:
But
- this sends ETH to `feeRecipient` no matter if the deposit is processed. It's just a donation to the fee recipient with no guarantee any work will be done.
- Deposit aren't processed faster/slower even in 3.1, we just wait for N number of block confirmations before making the amount available. So it's more of do/don't process, not a slower/faster one.

If we want to allow tipping the current system makes more sense. The fee is paid only when that specific deposit is processed, and it's probably even cheaper because this will do an additional ETH transfer for each deposit with tip.

Reply:
OK. updated.